relay-server/src/services/project/mod.rs

```diff
@@ -680,8 +594,21 @@ impl Project {
             return;
         }
 
+        let buckets = buckets.transform(metrics::PostProjectTransform {
```

Comment:
IMO it would be slightly easier to understand if this was a plain old standalone function, i.e.

`fn post_project_transform(buckets: Buckets<PreProject>) -> Buckets<PostProject>`

But there might be benefits to the transformer pattern that I'm not yet seeing.